clients/csrf/csrf_token_manager.go

```diff
@@ -0,0 +1,11 @@
+package csrf
+
+import "net/http"
+
+type CsrfTokenManager interface {
+	setCsrfToken() error
```

Comment:
Such naming conventions are great but they need to be follow by some method arguments to support them: setCsrfToken(<where-to-set-it>)

Reply:
Renamed to -> checkAndUpdateCsrfToken